PYME/localization/traveling_salesperson.py

```diff
@@ -84,7 +126,7 @@ def two_opt_test(route, i, k, distances, k_max):
 
 
 
-def two_opt_(distances, epsilon, initial_route=None):
+def two_opt(distances, epsilon, initial_route=None, fixed_endpoint=False):

```

Comment:
Gits merge seems to have done something gross when merging our alterations to this file - it ends up having two competing definitions of `two_opt()` with the same name (and possibly other issues).